### ISSUE TITLE
Add CodeCarbon to https://goodfirstissue.dev/

### ISSUE THREAD

**Issue by benoit-cty** [mlco2/codecarbon#486]
[https://goodfirstissue.dev/](https://goodfirstissue.dev/) is a web site to publish _good first issue_ automatically. 

View the process here https://github.com/DeepSourceCorp/good-first-issue#adding-a-new-project

**Comment by inimaz** [comment]
Good idea! I created a MR to their project here --> https://github.com/DeepSourceCorp/good-first-issue/pull/687

**Comment by benoit-cty** [comment]
The PR is still waiting...

**Comment by SaboniAmine** [comment]
Should we drop this issue ?

**Comment by inimaz** [comment]
I have just asked them in the PR, let's see if they reply.

**Comment by jacobboy** [comment]
This issue is resolved

<img width="631" height="605" alt="Image" src="https://github.com/user-attachments/assets/5944e5ce-c13a-4f38-bccd-59e74fd0e24e" />

**Comment by inimaz** [comment]
Thanks for the heads up @jacobboy ! Closing this then
